### PR TITLE
Feature/rails upgrade

### PR DESCRIPTION
**Pull request by Kalior**
Upgrades the rails version to 5.1, and adds support for not finding the lights.

### CODE REVIEW

File: app/controllers/lights_controller.rb [hunk 1]
```diff
@@ -167,6 +162,27 @@ def set_bulb_from_id
    end
 
    def set_lights
-      @lights = Huey::Bulb.all
+      begin
```
Comment (lindskogen) [inline app/controllers/lights_controller.rb]: Minor comment, `begin` is not required in ruby if the entire `begin-rescue` block starts at the beginning of a  method :)

**Review by lindskogen — COMMENTED** [review]
Nice idea with mocking the lights!
Maybe separate the MAJOR VERSION upgrade of rails into another PR next time :)

File: app/controllers/lights_controller.rb (hunk 1 above)
Comment (Kalior) [reply app/controllers/lights_controller.rb]: Really? Neat!

**Comment by Kalior** [comment]
Yeah, probably should have separated the upgrade into own PR... but I did that first and then I couldn't test it since I didn't have access to the lights and then I got lazy 🙇

**Comment by mattiasnilsen** [comment]
It is possible to create "real" Huey bulbs like this https://github.com/Veraticus/huey/blob/master/test/test_helper.rb#L27-L34

Don't know if that's better, but it's an option. Otherwise looks good 🗡

**Comment by Kalior** [comment]
@kerp1 nice, didn't spot that method. Will try that later

**Comment by Kerp1** [comment]
Got this error when running rake db:migrate
```
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateLockStates < ActiveRecord::Migration[4.2]
/home/kerp/projects/digIT/HueIT-Rails/db/migrate/20150730101331_create_lock_states.rb:1:in `<top (required)>'
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateLockStates < ActiveRecord::Migration[4.2]
/home/kerp/projects/digIT/HueIT-Rails/db/migrate/20150730101331_create_lock_states.rb:1:in `<top (required)>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

**Review by mattiasnilsen — CHANGES_REQUESTED** [review]
Got this error when running rake db:migrate
```
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateLockStates < ActiveRecord::Migration[4.2]
/home/kerp/projects/digIT/HueIT-Rails/db/migrate/20150730101331_create_lock_states.rb:1:in `<top (required)>'
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreateLockStates < ActiveRecord::Migration[4.2]
/home/kerp/projects/digIT/HueIT-Rails/db/migrate/20150730101331_create_lock_states.rb:1:in `<top (required)>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

**Review by mattiasnilsen — APPROVED** [review]

**Pull request merged**